web-example/src/hooks/useCobrowse.js

```diff
@@ -65,18 +65,33 @@ export const useCobrowse = () => {
       : undefined
 
     CobrowseIO.license = license ?? integrationLicense ?? 'trial'
-
-    CobrowseIO.redactedViews = Array.isArray(redactedViews) 
-      ? redactedViews 
-      : (typeof redactedViews === 'string' 
-        ? redactedViews.split(',').map(view => view.trim()) 
-        : ['.redacted', '#title', '#amount', '#subtitle', '#map'])
-
-    CobrowseIO.unredactedViews = Array.isArray(unredactedViews) 
-      ? unredactedViews 
-      : (typeof unredactedViews === 'string' 
-        ? unredactedViews.split(',').map(view => view.trim()) 
-        : ['.unredacted'])
+    
+    CobrowseIO.redactedViews = [
```

Comment:
I'd probably create a reusable method for merging the views as it's doing the same stuff three times. Just a nitpick though, up to you!